kleva-j/iReporter
# Feature/#163487402/update record from frontend

#### What does this PR do?
- Ensures a user can update a red-flag/intervention record from the frontend

#### Description of the task to be completed
 - create functions to make requests to the API from the frontend to update a red-flag/intervention record from the front-end

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#163487402/update-record-from-frontend
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[163487402](https://www.pivotaltracker.com/story/show/163487402)
